components/codec/src/columnar/mod.rs

```diff
@@ -134,6 +137,10 @@ struct ValuesEncoderImpl {
 /// The implementation for [`ValuesDecoder`].
 struct ValuesDecoderImpl;
 
+struct TimestampEncoderImpl;
```

Comment:
Move the `TimestampEncoderImpl` and `TimestampDecoderImpl` to the `timestamp.rs`.